.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   - "6"
 
-sudo: false
+sudo: required
```

Comment:
Issue described [here](https://github.com/travis-ci/travis-ci/issues/9024) and [here](https://github.com/travis-ci/travis-ci/issues/8836#issuecomment-356362524) on Travis' end.